boards/FSM/Inc/App/configs/App_BrakePressureSensorThresholds.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
+
+#define MIN_BRAKE_PRESSURE 0.0
+#define MAX_BRAKE_PRESSURE 2500.0f
```

Comment:
Adding units would be nice
```suggestion
#define MIN_BRAKE_PRESSURE_PSI 0.0
#define MAX_BRAKE_PRESSURE_PSI 2500.0f
```

Reply:
Ok nice. I'll add units to the flow meter and wheel speed constants too